### PR TITLE
[CBRD-22531]test json_extract with multi paths

### PR DESCRIPTION
**Pull request by mandy2016**
http://jira.cubrid.org/browse/CBRD-22531

### CODE REVIEW

File: sql/_31_cherry/issue_21672_json_functions/answers/json_path_2.answer [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+4     ["104",["ddd1","ddd2"]]     
+5     ["105",["ddd1","ddd2"]]     
+6     ["106","[\"eee1\",\"eee2\"]"]     
+7     ["107",{"name":"fff1","name_old":"fff2"}]     
+8     ["108",{"name":["ggg1","ggg2"],"name_old":"ggg3"}]     
+9     ["109",["hhh1",["hhh2","hhh3"],"hhh4","hhh5"]]     
+
+===================================================
+id    json_extract(c, '$.id')    json_extract(c, '$.id', '$.name[0]', '$.name[1]')    
+1     "101"     ["101",null,null]
```
[the remark this answers is not in the document]
Comment (jinmeihua) [reply sql/_31_cherry/issue_21672_json_functions/answers/json_path_2.answer]: I add the scenario for it.
http://jira.cubrid.org/browse/CBRD-22569?focusedCommentId=4752895&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-4752895

File: sql/_31_cherry/issue_21672_json_functions/answers/json_path_2.answer [hunk 2]
```diff
@@ -16,17 +16,15 @@ json_extract('[10, 20, [30, 40]]', '$[1]', '$[0]')
 ===================================================
 0
 ===================================================
-json_extract(@j1, '$.id', '$.name[0]', '$.name[1][1]', '$.name_old')    
-["109","hhh1","hhh3","hhh6"]     
-
+Error:-1200
```
Comment (jinmeihua) [inline sql/_31_cherry/issue_21672_json_functions/answers/json_path_2.answer]: I think it's a bug. Please open an issue. 
Test Build: 10.2.0.8389-9cec2c6
Actual Result:
```
csql> select json_extract('{"name_old":"hhh6"}', '$.name_old');                                                                                                                                                   
ERROR: Invalid JSON path


0 command(s) successfully processed. 
```
Expected Result:
Same as 
```
csql> select json_extract('{"name_old":"hhh6"}', '$."name_old"');

=== <Result of SELECT Command in Line 1> ===

  json_extract('{"name_old":"hhh6"}', '$."name_old"')
======================
  "hhh6"              

1 row selected. (0.009353 sec) Committed.

1 command(s) successfully processed.
```

File: sql/_31_cherry/issue_21672_json_functions/answers/json_path_2.answer [hunk 3]
```diff
@@ -16,17 +16,15 @@ json_extract('[10, 20, [30, 40]]', '$[1]', '$[0]')
...
 ===================================================
-json_extract(@j1, @j2, @j3, @j4, @j5)    
-["109","hhh1","hhh3","hhh6"]     
-
+Error:-1200
```
Comment (jinmeihua) [inline sql/_31_cherry/issue_21672_json_functions/answers/json_path_2.answer]: Same as above comment at line 19.

File: sql/_31_cherry/issue_21672_json_functions/answers/json_path_2.answer [hunk 4]
```diff
@@ -77,40 +77,30 @@ id    json_extract(c, '/id', '/name')
...
-2     "102"     ["102",null,null,null,null]     
-3     "103"     ["103",null,null,null,null]     
-4     "104"     ["104",null,null,null,null]     
-5     "105"     ["105",null,null,null,null]     
-6     "106"     ["106",null,null,null,null]     
-7     "107"     ["107","fff1",null,null,"fff2"]     
-8     "108"     ["108",["ggg1","ggg2"],"ggg1","ggg2","ggg3"]     
-9     "109"     ["109",null,null,null,null]     
-
+Error:-1200
```
Comment (jinmeihua) [inline sql/_31_cherry/issue_21672_json_functions/answers/json_path_2.answer]: Same as the above comment at line 19.

**Comment by jinmeihua** [comment]
@mandy2016 
I don't agree with the fix. 
Please open an issue for 'Error:-1200'.

**Comment by mandy2016** [comment]
report: CBRD-23072
I will merge the PR after CBRD-23072 is fixed

**Pull request merged**